index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">  
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+   <div class="topnav">
+    <a class="logo" href="#">Bobson</a>
+       <nav class ='hambuger'>
+        <div class ="hambuger-bar"></div>
+        <div class ="hambuger-bar"></div>
+        <div class ="hambuger-bar"></div>
+       </nav>
+       </div>
+  </header>
+  <main class="main-body">
+<section class ="container ">
+  <div class ="first-section" >
+    <div>
+    <h1 class ="title">
+      I'm Bobson <br> Glad to see you!
+    </h1>
+    <p class="intro">
+      I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <h3 class = "getintouch">
+      LET'S CONNECT
+    </h3>
+    </div>
+    <ul class="social-media-icon" >
+     <li><a class = "social-media" href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+     <li> <a class = "social-media"href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
+     <li><a class = "social-media"href="#"><img src="./images/medium.png" alt="medium"></a> </li>
+     <li><a class = "social-media" href="#"><img src="./images/github.png" alt="github"></a></li>
+     <li><a class = "social-media"href="#"><img src="./images/anglelist.png"alt="anglelist"></a></li>
+ </ul>
+ </div>
+</section>
+<section class="section">
+<section class="second-section">
+  <div class="section-2">
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio.png" alt="picture" >
+    </div>
+    <div>
+      <h2 class="proj-title">Tonic</h2>
+    </div>
+    <div class="proj-intro">
+      <span class="intro-1">CANOPY</span>
```

Comment:
- [ ] The project title (e.g., "Tonic" and "CANOPY") should use a header tag `hx`. Kindly make the change to follow best semantic practices 👍 
- Kindly do so for all other projects 👍